week-4/BookLibrary/Library.sol

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.17;
+
+import "./BookRepository.sol";
+
+// Library contract is the main contract for the library
+contract Library is BookRepository {
+    // Membership fee for opening account
+    uint256 fee = 5 ether;
+
+    // Books that are borrowed by a member
+    struct Books {
```

Comment:
Instead of creating another ```Books```, could have used the  ```bookRecords``` to manage the borrowing as it already contains all the necessary variables.